halo2_proofs/Cargo.toml

```diff
@@ -0,0 +1,75 @@
+[package]
+name = "halo2_proofs"
+version = "0.1.0-beta.1"
```

Comment:
I left this at the same version as `halo2` is currently published as, because I think it will be clearer if `halo2_proofs 0.1.0-beta.2` is the follow-on release from `halo2 0.1.0-beta.1`.